package/ios/RNSkia-iOS/SkiaDrawView.mm

```diff
@@ -61,7 +61,7 @@ - (void) willMoveToWindow:(UIWindow *)newWindow {
   } else {
     // Create implementation view when the parent view is set
     if(_impl == nullptr && _manager != nullptr) {
-      __weak typeof(self) weakSelf = self;
+      __weak decltype(self) weakSelf = self;
```

Comment:
Nice catch! Was looking into this one after I started building against Fabric earlier today!!!

Reply:
@chrfalch  did you manage to build project for Fabric?